intro-Linux/ch_getting_started.tex

```diff
@@ -154,10 +170,17 @@ \subsection{Using non-defined variables}
 %\homedir%
 \end{prompt}
 
-To understand what's going on here, see the section on ``cd'' below.
+To understand what's going on here, see the section on \verb|cd| below.
 
 The moral here is: \strong{be very careful to not use empty variables unintentionally}.
 
+\strong{Tip for job scripts: use \texttt{set -e -u} to avoid using empty variables accidentally.}
```

Comment:
Either use `\verb` or `\texttt`, don't mix both (or explain why you're mixing them).

Also open a separate PR for the style guide on using `\verb` or `'\texttt`

Reply:
I'll put it in the styleguide later, but short explanation: \verb can't be used in arguments (so not in strong, section, ...). \verb is better though, because you don't have to worry about escaping double hyphens,...

Reply:
Ah, you already did (#217), thanks.